examples/pspnet/caffe2npz.py

```diff
@@ -0,0 +1,251 @@
+from __future__ import print_function
```

Comment:
I think this script should be more readable.

Reply:
OK